libmozevent/utils.py

```diff
@@ -146,6 +146,12 @@ class AsyncRedis(object):
     async def connect():
         if AsyncRedis.redis.get(None) is None:
             AsyncRedis.redis.set(
-                await aioredis.from_url(os.environ["REDIS_URL"], decode_responses=False)
+                await aioredis.from_url(
+                    os.getenv("REDIS_TLS_URL", None)
+                    if os.getenv("REDIS_SSL_URL", None) is not None
+                    else os.getenv("REDIS_URL", None),
```

Comment:
It could be better for readability if this got assigned to a variable first.

Reply:
I will have to do it in another way because it doesn't support `ssl_cert_reqs `